### ISSUE TITLE
Unable to 'add' both Ruby 2.0 mingw64 and mingw32

### ISSUE THREAD

**Issue by zhon** [vertiginous/pik#148]
When I install either mingw32 or mingw64, it overwrites the other one using the same name

ruby-2.0.0-p0

**Comment by luislavena** [comment]
Hello @zhon, this is a known limitation of current Pik.

I suggest you manually download the x64 package, extract it and use `pik add` to add the other version.

For example, I manually install all my rubies, this is how my configuration looks like:

``` yaml
"[ruby-]2.0.0-p0": 
  :alias: "200"
  :path: !ruby/object:Pathname 
    path: C:/Users/Luis/Tools/Ruby/ruby-2.0.0-p0-i386-mingw32/bin
  :version: |
    ruby 1.9.3p392 (2013-02-22) [i386-mingw32]

"[ruby-]2.0.0-p0-x64": 
  :alias: 200-64
  :path: !ruby/object:Pathname 
    path: C:/Users/Luis/Tools/Ruby/ruby-2.0.0-p0-x64-mingw32/bin
  :version: |
    ruby 2.0.0p0 (2013-02-24) [x64-mingw32]
```

**Comment by zhon** [comment]
After using pik add, I ended up with the same name. So after 'pik add <path>' you must change the config file?

Where is the pik config file so I can edit it?

**Comment by luislavena** [comment]
> So after 'pik add ' you must change the config file?

Yeah, I ended editing the file manually and adding `alias` key to allow me easy reference it.

> Where is the pik config file so I can edit it?

`%USERPROFILE%\.pik\config.yml`
